.github/workflows/_run-node-cypress-tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Github Actions for NodeJS apps - run cypress tests
+
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  cypress-tests:
+    runs-on: ubuntu-latest
+    name: Cypress tests
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b
+      - name: Parse Node version
```

Comment:
You can actually ditch this, in the setup task for actions/setup-node you can do:

```
with:
   node-version-file: '.nvmrc'
```